src/test/moment/duration.js

```diff
@@ -53,10 +53,17 @@ test('milliseconds instantiation', function (assert) {
 
 test('undefined instantiation', function (assert) {
     assert.equal(moment.duration(undefined).milliseconds(), 0, 'milliseconds');
+    assert.equal(moment.duration(undefined)._isValid, true, '_isValid');
 });
 
 test('null instantiation', function (assert) {
     assert.equal(moment.duration(null).milliseconds(), 0, 'milliseconds');
```

Comment:
This doesn't really fix the #3225 because `humanize` will still output "in a few seconds".